### DISCUSSION TITLE
Mullvad / QBittorrent Port Forwarding in docker on Synology 920+

### DISCUSSION THREAD

**Discussion by Nutmax** [qdm12/gluetun#1331] [Q&A]
Hello I could use some help with port forwarding! I am a bit new to this and getting to this point as been truly a task for me. I'm at a point where I can't figure it out myself. 

I am running Qbittorent and Gluetun in a stack in docker in my Synology 920+. I have been able to successfully connect Qbittorent to only run when Gluetun is working. I am using Mullvad and tried to understand port forwarding but I think I am confused and not understanding something.  

I went with the OPEN VPN option in the guide. 

One thing I don't understand is: In my Mullvad account page, I only have 2 devices active, which is my desktop PC (Crusty Crab) and my Phone (Salty Elephant) so I don't know how to get the port to forward to my Qbittorent in my NAS. 

I hope I'm explaining this correctly. Please let me know if I can clarify anything. I tried searching for an answer but the closest thing I got was this post: https://github.com/qdm12/gluetun/discussions/1172

So my question is about finding out how to port forward Mullvad for Qbittorent that are in docker on my synology NAS. Thanks so much!

qbittorrent:
    image: linuxserver/qbittorrent
    container_name: qbittorrent
    environment:
      - PUID=1027
      - PGID=100
      - TZ={TZ}
      - WEBUI_PORT=8090
    volumes:
      - ${DOCKERCONFDIR}/qbittorrent:/config:rw
      - ${DOCKERSTORAGEDIR}/torrents:/data/torrents:rw
    network_mode: service:gluetun # run on the vpn network
    depends_on:
      - gluetun
    restart: unless-stopped
    
  gluetun:
    image: qmcgaw/gluetun
    container_name: gluetun
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun:/dev/net/tun
    ports:
      - 8888:8888/tcp # HTTP proxy
      - 8388:8388/tcp # Shadowsocks
      - 8388:8388/udp # Shadowsocks
      - 8090:8090 # port for qbittorrent
      - 6881:6881 #qbittorrent listening port
    volumes:
      - ${DOCKERCONFDIR}/gluetun:/config:rw
    environment:
      - VPN_SERVICE_PROVIDER=mullvad
      - VPN_TYPE=openvpn
      - OPENVPN_USER=MyMullvadAccountIDHere
      - SERVER_CITIES=MyCityHere
      - FIREWALL_VPN_INPUT_PORTS=
      - TZ={TZ}
      - HTTPPROXY=on #change to off if you don't wish to enable
      - SHADOWSOCKS=on #change to off if you don't wish to enable
    network_mode: trash-guides_default
    restart: unless-stopped

**Comment by frasderp** [comment]
In Mullvad, where you select the Port Forwarding options, there is an option for "No Device (only OpenVPN).", thats what you should use for your setup.
